tests/distributed_grids/1d_grid.cc

```diff
@@ -23,16 +23,6 @@
 
 int main(int argc, char *argv[])
 {
-  deal_II_exceptions::disable_abort_on_exception();
   Utilities::MPI::MPI_InitFinalize mpi_initialization (argc, argv, 1);
-  initlog();
-
-  try
-    {
-      parallel::distributed::Triangulation<1> tr(MPI_COMM_WORLD);
-    }
-  catch (const std::exception &exc)
-    {
-      deallog << exc.what() << std::endl;
```

Comment:
If the expectation for this test is to produce an exception, can't you just output here "Exception caught" or similar to make sure that the test continues to throw an exception but work otherwise?